src/Consolonia.Core/Drawing/BrushExtensions.cs

```diff
@@ -84,9 +85,13 @@ public static Color FromPosition(this IBrush brush, int x, int y, int width, int
                 case ISolidColorBrush solidColorBrush:
                     return solidColorBrush.Color;
 
+                case ShadeBrush:
+                case BrightenBrush:
+                case InvertBrush:
+                    return Colors.Transparent;
                 default:
                     ConsoloniaPlatform.RaiseNotSupported(751, brush); //todo: allow RaiseNotSupported to return a result
-                    return new Color();
+                    return Colors.Transparent;
```

Comment:
In the future I want RaiseNotSupported to be able to return a result. So if something we are not supporting to draw, the user will be able to override. It's just idea.